pom.xml

```diff
@@ -222,6 +222,8 @@
 
     <slf4j.version>2.0.7</slf4j.version>
     <junit-bom.version>5.11.2</junit-bom.version>
+    <mockito.version>4.3.1</mockito.version>
```

Comment:
Let's also update the versions
- junit: 5.12.2
- mockito: 4.11.0 (upgrading to mockito 5 seems not easy: https://davidvlijmincx.com/posts/upgrade-to-mockito-5/)